dashboard/modules/actor/actor_head.py

```diff
@@ -56,20 +56,17 @@ def actor_table_data_to_dict(message):
         "state",
         "name",
         "numRestarts",
-        "taskSpec",
+        "functionDescriptor",
         "timestamp",
         "numExecutedTasks",
     }
     light_message = {k: v for (k, v) in orig_message.items() if k in fields}
-    if "taskSpec" in light_message:
-        actor_class = actor_classname_from_task_spec(light_message["taskSpec"])
```

Comment:
please delete `actor_classname_from_task_spec` if it's no longer used anywhere

Reply:
We just don't use it here. It's still used by other codes.